js/app.js

```diff
@@ -8,6 +8,163 @@ const menuBody = document.getElementById('mobile-menu-body');
 const menuBlur = document.getElementById('mobile-menu-blur');
 const menuHeaderTop = document.getElementById('menu-header-top');
 const menuLogo = document.getElementById('menu-logo');
+const tonicProjectModal = document.getElementById('modal-div');
+const tonicProject = document.getElementById('tonic-project-btn');
+const multipostProject = document.getElementById('multipost-project-btn');
+const tonicProject2 = document.getElementById('tonic-project-2-btn');
+const multipostProject2 = document.getElementById('multipost-project-2-btn');
+const errorDiv = document.getElementById('error-output');
+const form = document.getElementById('contact_form');
+
+form.addEventListener('submit', (e) => {
+  const email = document.getElementById('email').value;
+  if (email !== email.toLowerCase()) {
+    errorDiv.innerHTML = 'Oops! the email should be in lowercase';
```

Comment:
- [ ] Kindly ensure that the error message should show up close to the "submit" button and not at the top of the contract section as specified in the project requirement 👍🏽 .